test/controllers/setupquestions/lifetimeallowance/ChangeInLifetimeAllowanceControllerSpec.scala

```diff
@@ -79,7 +79,7 @@ class ChangeInLifetimeAllowanceControllerSpec extends SpecBase with MockitoSugar
       }
     }
 
-    "must redirect to the ChangeInTaxCharge page when valid data is submitted" in {
+    "must redirect to the ChangeInTaxCharge page when true is submitted" in {
```

Comment:
Don't we just need to check for the 303 status here?

Reply:
This applies to other controller specs as well.

Reply:
Done